ajax/libs/jquery-ajax-localstorage-cache/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery-ajax-localstorage-cache",
+  "filename": "jalc.min.js",
+  "description": "Store ajax responses using the browser's local storage, or any Storage-interface compatible object.",
+  "version": "2.0.1",
+  "keywords": [
+    "ajax",
+    "localstorage",
+    "caching"
+  ],
+  "author": "SaneMethod",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SaneMethod/jquery-ajax-localstorage-cache"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/SaneMethod/jquery-ajax-localstorage-cache.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
```

Comment:
Let's also add `jquery-ajax-localstorage-cache.js` which is non-minified file of `jalc.min.js`.
https://github.com/SaneMethod/jquery-ajax-localstorage-cache/blob/master/gruntFile.js#L20-L21

```js
      {
        "basePath": "",
        "files": [
          "jquery-ajax-localstorage-cache*"
        ]
      }
```

The whole auto-update config is 
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/SaneMethod/jquery-ajax-localstorage-cache.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      },
      {
        "basePath": "",
        "files": [
          "jquery-ajax-localstorage-cache*"
        ]
      }
    ]
  }
```
Thank you.